sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-multibinders/src/main/java/com/azure/spring/sample/eventhubs/multibinders/EventProducerController.java

```diff
@@ -33,13 +33,13 @@
     public ResponseEntity<String> sendMessage1(@RequestParam String message) {
         LOGGER.info("Going to add message {} to Sinks.Many1", message);
```

Comment:
how about change the log info , "Sinks.Many1"-->to  the method name "sendMessage1"

Reply:
OK